tests_metricflow/snapshots/test_fill_nulls_with_rendering.py/SqlQueryPlan/DuckDB/test_join_to_time_spine_with_filters__plan0_optimized.sql

```diff
@@ -10,22 +10,28 @@ FROM (
   -- Join to Time Spine Dataset
   -- Constrain Time Range to [2020-01-03T00:00:00, 2020-01-05T00:00:00]
   SELECT
-    subq_17.metric_time__day AS metric_time__day
-    , subq_16.bookings AS bookings
+    subq_23.metric_time__day AS metric_time__day
+    , subq_18.bookings AS bookings
   FROM (
-    -- Filter Time Spine
+    -- Constrain Output with WHERE
+    -- Constrain Time Range to [2020-01-03T00:00:00, 2020-01-05T00:00:00]
+    -- Pass Only Elements: ['metric_time__day',]
     SELECT
       metric_time__day
     FROM (
       -- Read From Time Spine 'mf_time_spine'
+      -- Transform Time Dimension Columns
       SELECT
         ds AS metric_time__day
         , DATE_TRUNC('week', ds) AS metric_time__week
-      FROM ***************************.mf_time_spine subq_18
-      WHERE ds BETWEEN '2020-01-03' AND '2020-01-05'
-    ) subq_19
-    WHERE metric_time__week > '2020-01-01'
-  ) subq_17
+      FROM ***************************.mf_time_spine time_spine_src_28006
+    ) subq_20
+    WHERE (
```

Comment:
filter now in outer query, makes sense with alias change, but why did this change?